examples/rust/opensk/Cargo.toml

```diff
@@ -7,13 +7,12 @@ edition = "2021"
 
 [dependencies]
 opensk-lib = { path = "../../../third_party/google/OpenSK/libraries/opensk", package = "opensk" }
-persistent_store = { path = "../../../third_party/google/OpenSK/libraries/persistent_store" }
 rand_core = "0.6.4"
 wasefire = { path = "../../../crates/prelude" }
-wasefire-stub = { path = "../../../crates/stub", optional = true }
+wasefire-error = { path = "../../../crates/error" }
 
 [build-dependencies]
 uuid = "1.8.0"
 
 [features]
-test = ["dep:wasefire-stub", "opensk-lib/std", "persistent_store/std", "wasefire/test"]
```

Comment:
Actually, you might need `wasefire-stub` for the tests. But let's see what the CI says. I don't understand why it's not running.

Reply:
Ok, I've figured it out. I forgot to update the workflow when I renamed the branch. I'll update the PR to see if it works.